README.md

```diff
@@ -159,6 +159,13 @@ The pipeline should be executed as follows:
 
 Note: [Module 00c](#module00c) requires a [trained gCNV model](#gcnv-training).
 
+#### <a name="batching">Batching</a>
+For larger cohorts, samples should be split up into batches of about 100-500 samples with similar characteristics. We recommend batching based on overall coverage and dosage score (WGD), which can be generated in [Module 00b](#module00b). An example batching process is outlined below:
+1. Divide the cohort into PCR+ and PCR- samples
+2. Partition the samples by median coverage from [Module00b](#module00b)
```

Comment:
This is a little confusing to me since without having done it before I wouldn't be sure of how to partition based on median coverage -- how many partitions would I want to create? Would it depend on the shape of the distribution of median coverages in my cohort? Perhaps if you move the guidance about the desired number of samples you want in each batch from the next point earlier it would help?